src/commands/environmentVariables/deleteEnvironmentVariable/EnvironmentVariableDeleteContext.ts

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { type EnvironmentVar } from "@azure/arm-appcontainers";
+import { type SetTelemetryProps } from "../../../telemetry/SetTelemetryProps";
+import { type ContainerUpdateTelemetryProps as TelemetryProps } from "../../../telemetry/commandTelemetryProps";
+import { type EnvironmentVariablesBaseContext } from "../EnvironmentVariablesContext";
+
+export interface EnvironmentVariableDeleteBaseContext extends EnvironmentVariablesBaseContext {
+    environmentVariable: EnvironmentVar;
```

Comment:
Should this just be part of the edit context?

Reply:
Just to clarify, which edit context are you referring to?

Reply:
There is an interface called `EnvironmentVariableEditBaseContext ` or something like that. But it is shaped very similarly to this interface.

Reply:
Oh I see what you mean, yeah they are super similar, I think the notable difference is the base interface being extended from.  I didn't want to include all the editable properties on the context for the delete command